### DISCUSSION TITLE
openshell uninstaller almost crashed my windows

### DISCUSSION THREAD

**Discussion by ikkqqstudio** [Open-Shell/Open-Shell-Menu#2001] [General]
### Describe the bug

today, when I uninstall openshell, and I restart my windows, my pc goes blank and the cursor remaining, I have no idea how to fix this, I reinstalled Windows

### Area of issue

Installation/Other

### To reproduce

idk

### Expected behavior

idk

### Open-Shell version

newer

### Windows version

Windows​ 10 22H2

**Comment by ge0rdi** [comment]
Try to boot into safe-mode and manually remove eventual `Open-Shell` remnants using our [Utility](https://github.com/Open-Shell/Open-Shell-Menu/releases/download/v4.4.191/Utility.exe).
